content/courses/program-optimization/lookup-tables.md

```diff
@@ -5,65 +5,81 @@ objectives:
   - Create lookup tables
   - Extend lookup tables
   - Use lookup tables with versioned transactions
-description: "Use large amounts of accounts by using lookup tables."
+description:
+  "Harness a great deal of accounts through the application of lookup tables."
```

Comment:
Honestly the old one was simpler. Plain language is important. 

1. Not everyone speaks english. Not everyone knows the word 'harness'. 
2. People want simple clear directions. It should be clear that if they need a lot of accounts in their translations, they should use lookup tables.

> "Use large amounts of accounts in a transaction by using lookup tables."